.openpublishing.redirection.json

```diff
@@ -114,6 +114,11 @@
         "redirect_url": "/microsoft-edge/devtools-guide-chromium/evaluate-performance/",
         "redirect_document_id": false
       },
+      {
+        "source_path": "./microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks.md",
+        "redirect_url": "/microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks-memory-tool-detached-elements",
+        "redirect_document_id": false
+      },
```

Comment:
Where do you end up when go to this link?
https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks?branch=pr-en-us-3399
@captainbrosset

Reply:
This now ends up at: **Debug DOM memory leaks ("Detached elements" profiling type)** (https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks-memory-tool-detached-elements?branch=pr-en-us-3399)